guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -29,8 +25,7 @@ The advantage of this is that each organization maintains separate subscriptions
 You can use future-dated subscriptions in a subscription allocation.
```

Comment:
```suggestion
You can use future-dated subscriptions in a subscription manifest.
```

On console.redhat.com they just call them "Manifests;" I think "subscription allocation" is the older term used on the deprecated access.redhat.com.